### PR TITLE
Ne Prisma Cloud Compute Audit Playbook

### PR DESCRIPTION
**Pull request by nericksen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR introduces a new playbook, layout, etc for handling WaaS Audit alerts generated in Prisma Cloud Compute. It is designed to be a general response handling playbook.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by nericksen** [comment]
Working through the formatting of this now that the new commands to the integration were pushed.

**Comment by adi88d** [comment]
@nericksen Please fille the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form

**Comment by nericksen** [comment]
> @nericksen Please fille the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form

filled out thanks.  I have some more linting and formatting to do on this one I will work on tomorrow to complete.

**Comment by adi88d** [comment]
Thanks @nericksen, let me know when can i start the review

**Comment by adi88d** [comment]
Hi @nericksen ,
Haven’t heard from you since last week.
Do you need any help with something?
Please don’t hesitate to reach out to me here or on Slack.
Thanks again.

**Comment by nericksen** [comment]
Apologies, will finish this up today.

**Comment by nericksen** [comment]
Apologies, will finish this up today.

**Comment by nericksen** [comment]
@adi88d Release notes are now updated, test playbook updated, linting and validation handled.

One question I did have was how to resolve this validation error on the layout: "Those types are not supported in XSIAM, remove them or change the layout to be XSOAR only" ..

Let me know your additional feedback and I will work to implement.

File: Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml [hunk 1]
```diff
@@ -839,6 +837,12 @@ script:
       required: true
       isArray: true
       defaultValue: ""
+    - default: true
+      description: ''
+      isArray: true
+      name: cve
+      required: true
+      secret: false
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml:845]: why you added this argument?
```suggestion
    - default: true
      description: ''
      isArray: true
      name: cve
      required: true
      secret: false
```

File: Packs/PrismaCloudCompute/IncidentTypes/incidenttype-Prisma_Cloud_Compute_Audit_v2.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
+        "fieldCliNameToExtractSettings": {},
+        "mode": "All"
+    },
+    "hours": 0,
+    "hoursR": 0,
+    "id": "Prisma Cloud Compute Audit v2",
+    "layout": "247e2761-5de0-47e8-8bd3-7b22ec91fd51",
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/IncidentTypes/incidenttype-Prisma_Cloud_Compute_Audit_v2.json:16]:
```suggestion
    "layout": "Prisma Cloud Compute Audit Layout v2",
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 3]
```diff
@@ -0,0 +1,1104 @@
...
+        simple: '${PrismaCloudCompute.AuditAlert.category} '
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 69a6d29a-a700-4b45-8336-2bbfb6bab7a0
+      iscommand: false
+      name: Create Link Back
+      description: Create a link back to the Prisma Cloud Compute instance.
+      script: 1c969de3-c684-4977-872b-295ad7be3f0b
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:998]: in this task you use the `CreatePrismaCloudComputeLink` script
please use the HttpV2 script instead and try to use in without hardcoded values

File: Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json [hunk 4]
```diff
@@ -0,0 +1,955 @@
...
+                                "buttonClass": "success",
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "e6bd95d0-fe17-11ec-8186-47081b53acb3",
+                                "index": 1,
+                                "listId": "caseinfoid-d8e8a170-fe17-11ec-8186-47081b53acb3",
+                                "name": "Tag IP or Domain",
+                                "scriptId": "5adcc0b4-c7db-4579-8049-4857568bfb4f",
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json:421]:
```suggestion
                                "scriptId": "TagIndicatorButton",
```

File: Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json [hunk 5]
```diff
@@ -0,0 +1,955 @@
...
+                                    }
+                                },
+                                "buttonClass": "primary",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "fafce270-fe18-11ec-8186-47081b53acb3",
+                                "index": 4,
+                                "name": "Email CVE Report",
+                                "scriptId": "27a08129-1f06-4b12-8a24-e585fc2d4fd8",
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json:469]:
```suggestion
                                "scriptId": "CreatePrismaCloudComputeComplianceReportButton",
```

File: Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json [hunk 6]
```diff
@@ -0,0 +1,955 @@
...
+                                        "simple": "Prisma  Cloud Compute CVEs"
+                                    }
+                                },
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "b7143db0-fe18-11ec-8186-47081b53acb3",
+                                "index": 0,
+                                "name": "Email CVE Report",
+                                "scriptId": "27a08129-1f06-4b12-8a24-e585fc2d4fd8",
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json:795]:
```suggestion
                                "scriptId": "CreatePrismaCloudComputeComplianceReportButton",
```

File: Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.py [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import demistomock as demisto  # noqa: F401
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.py:1]: - please add **main** method
- don't use demisto.results() Use return_results() instead
- surround the code with **try** **except** block
- do the same for **CreatePrismaCloudComputeComplianceReportButton**

File: Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.py [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+from CommonServerPython import *  # noqa: F401
+args = demisto.args()
+res = demisto.executeCommand("setIndicators", {"indicatorsValues": args.get("Indicators"), "tags": args.get("Tags")})
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.py:4]: please do the following in all the places you used **demisto.executeCommand**
```suggestion
res = demisto.executeCommand("setIndicators", {"indicatorsValues": args.get("Indicators"), "tags": args.get("Tags")})
if is_error(res):
    raise DemistoException(f'Failed to set indicators: {str(get_error(res))}')

```

**Review by adi88d — CHANGES_REQUESTED** [review]
@nericksen Thank you for your contribution!
Good work :)
please revert the following files:
- layoutscontainer-Prisma_Cloud_Compute_Compliance.json
- PaloAltoNetworks_PrismaCloudCompute.yml
- PrismaCloudComputeParseAuditAlert.yml + README.md
- PrismaCloudComputeParseCloudDiscoveryAlert.yml + README.md
- PrismaCloudComputeParseComplianceAlert.yml + README.md
- PrismaCloudComputeParseVulnerabilityAlert.yml + README.md

see my other comments below

**Comment by adi88d** [comment]
> Those typ

im checking about it for you

**Comment by nericksen** [comment]
Thank you @adi88d will start working through this!

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml (hunk 3 above)
Comment (nericksen) [reply Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:998]: whoops! yes will fix the hardcoded values in `CreatePrismaCloudComputeLink`. This doesn't actually make the HTTP request to the URL it just creates the format of the link back to alerts for the image in Prisma Cloud Compute. Will fix this one.

**Comment by nericksen** [comment]
Will work through the rest of automation updates on Wednesday evening EST.

**Comment by nericksen** [comment]
Ok I think now I just have to test the `cve` issue you mentioned above and the `PCC` field error mentioned in CI/CD output. Scripts should be good to go.

Then will do another round of full end to end testing.

**Comment by nericksen** [comment]
@adi88d lmk your thoughts on this.  All feedback should now be implemented. Thank you as always for your comments and patience :)

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml (hunk 3 above)
Comment (adi88d) [reply Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml]: in that case please create a task in the playbook to create the url using **concat** transformer

File: Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json (hunk 4 above)
Comment (adi88d) [reply Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json]: i dont see where you using **TagIndicatorButton** script in the playbook

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.py [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    Hello,
+
+    Please see below the details for the compliance report from Prisma Cloud Compute
+
+    {html}
+
+    - DBot
+    """
+
+    try:
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.py:27]: no need to use **try except** block twice
remove this please

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.py [hunk 10]
```diff
@@ -0,0 +1,42 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+def main():
+    args = demisto.args()
+
+    try:
+        res = demisto.executeCommand("ConvertTableToHTML", {"table": args.get("table"), "title": args.get("title")})
+
+        if is_error(res):
+            raise DemistoException(f'Failed to set indicators: {str(get_error(res))}')
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.py:10]:
```suggestion
            raise DemistoException(f'Failed to create compliance report: {str(get_error(res))}')
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.py [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+    {html}
+
+    - DBot
+    """
+
+    try:
+        res = demisto.executeCommand("send-mail", {"to": args.get("to"), "subject": "IMPORTANT: Prisma Cloud Compute Compliance", "body": body})
+
+        if is_error(res):
+            raise DemistoException(f'Failed to set indicators: {str(get_error(res))}')
```
Comment (adi88d) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.py:31]:
```suggestion
            raise DemistoException(f'Failed to send  the compliance report: {str(get_error(res))}')
```

**Review by adi88d — CHANGES_REQUESTED** [review]
please generate README for the scripts

**Comment by efelmandar** [comment]
@nericksen Hi and thank you for your time and contribution, I reviewed the PR and overall it looks good! 
One thing to emphasize, please note that you are using incident fields in your new playbook, which are *not* associated with your new incident type. This might cause an issue in the playbook since they won't exist in the incident content.
To fix this - just associate those fields `prisma cloud compute fqdn`, `prisma cloud compute image`, `prisma cloud compute host`, `prisma cloud compute raw alert json` with your new incident type `Prisma Cloud Compute Audit v2`.

**Comment by nericksen** [comment]
@efelmandar thanks and good catch!  I just updated the fields to include the new incident type.

@adi88d READMEs have now been generated for the scripts as well.

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 12]
```diff
@@ -0,0 +1,1094 @@
...
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: b4fd8f72-715b-441a-84c9-dbd6efb3ad7f
+    type: regular
+    task:
+      id: b4fd8f72-715b-441a-84c9-dbd6efb3ad7f
+      version: -1
+      name: Get Defender Policy
+      description: DefenderSettings returns the defenders settings
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:300]:
```suggestion
      description: DefenderSettings returns the defenders settings.
```
Maybe this description should be something like:
DefenderSettings returns the settings in the Defender policy.

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 13]
```diff
@@ -0,0 +1,1094 @@
...
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 1eb223af-dfa4-4883-8254-f6e92444bad9
+    type: regular
+    task:
+      id: 1eb223af-dfa4-4883-8254-f6e92444bad9
+      version: -1
+      name: Get Alert Profiles
+      description: AlertProfiles returns the all alert profiles
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:333]:
```suggestion
      description: AlertProfiles returns all the alert profiles.
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 14]
```diff
@@ -0,0 +1,1094 @@
...
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: 9ef59bcc-e809-4cfc-8f13-21319879dc2e
+    type: regular
+    task:
+      id: 9ef59bcc-e809-4cfc-8f13-21319879dc2e
+      version: -1
+      name: List Defender Backups
+      description: List returns the available backups
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:517]:
```suggestion
      description: List returns the available backups.
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 15]
```diff
@@ -0,0 +1,1094 @@
...
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 778b86c5-6e7b-457a-80f1-63031dceb702
+    type: regular
+    task:
+      id: 778b86c5-6e7b-457a-80f1-63031dceb702
+      version: -1
+      name: Download Defender Logs
+      description: 'DownloadDefenderLogs downloads all defender logs and metadata '
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:550]:
```suggestion
      description: DownloadDefenderLogs downloads all defender logs and metadata.
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 16]
```diff
@@ -0,0 +1,1094 @@
...
+  "38":
+    id: "38"
+    taskid: 357a1a54-9d3c-4cc7-811b-a7c8663f3ed9
+    type: regular
+    task:
+      id: 357a1a54-9d3c-4cc7-811b-a7c8663f3ed9
+      version: -1
+      name: Get container forensics
+      description: Get runtime forensics data for a specific container on a specific host
+      tags:
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:703]:
```suggestion
      tag:
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 17]
```diff
@@ -0,0 +1,1094 @@
...
+    isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 2b732479-d128-4249-8585-505e4e857ceb
+    type: regular
+    task:
+      id: 2b732479-d128-4249-8585-505e4e857ceb
+      version: -1
+      name: Get Image Audits
+      description: Get the audits for the firewall container policies
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:780]:
```suggestion
      description: Get the audits for the firewall container policies.
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 18]
```diff
@@ -0,0 +1,1094 @@
...
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 3d3710fa-7779-4295-8c5f-6620ea78f8fc
+    type: regular
+    task:
+      id: 3d3710fa-7779-4295-8c5f-6620ea78f8fc
+      version: -1
+      name: Set Audits to Table
+      description: Convert an array to a nice table display. Usually, from the context.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:822]:
```suggestion
      description: Convert an array to a nice table display (usually from the context).
```

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
+
+#### Scripts
+
+##### New: CreatePrismaCloudComputeComplianceReportButton
+
+- Generate a compliance report via button.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:6]:
```suggestion
Generate a compliance report via button.
```

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+
+##### New: CreatePrismaCloudComputeLink
+
+- This script creates a link back to the Prisma Cloud Compute instance.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:10]:
```suggestion
This script creates a link back to the Prisma Cloud Compute instance.
```

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+
+##### New: TagIndicatorButton
+
+- This is a wrapper around the setIndicators script
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:14]:
```suggestion
This is a wrapper around the setIndicators script.
```

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+
+##### New: Prisma Cloud Compute Audit Layout v2
+
+#### Incident Fields
+
+- New: **Prisma Cloud Compute Defender Logs**
+
+- New: **Prisma Audit Table**
+
+- **Prisma Cloud Compute Category**
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:27]: Is this a new incident field?  An updated incident field?

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+
+- New: **Prisma Cloud Compute Defender Logs**
+
+
+#### Incident Types
+
+- New: **Prisma Cloud Compute Audit v2**
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:34]:
```suggestion
New: **Prisma Cloud Compute Audit v2**
```

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+
+
+#### Playbooks
+
+##### New: Prisma Cloud Compute - Audit Alert v2
+
+- Default playbook for parsing Prisma Cloud Compute audit alerts
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:41]:
```suggestion
Default playbook for parsing Prisma Cloud Compute audit alerts
```

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+
+
+#### Integrations
+
+##### Palo Alto Networks - Prisma Cloud Compute
+
+- Updated the Docker image to: *demisto/python3:3.10.10.48392*.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:48]:
```suggestion
Updated the Docker image to: *demisto/python3:3.10.10.48392*.
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml [hunk 26]
```diff
@@ -0,0 +1,28 @@
+args:
+- isArray: true
+  name: table
+  required: true
+  description: The table of data.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml:5]: Is this a comma-separated list?   I see that isArray is true.

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml [hunk 27]
```diff
@@ -0,0 +1,28 @@
...
+- name: title
+  required: true
+  description: The title of the report.
+- description: The to email
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml:9]:
```suggestion
- description: The to email address.
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md [hunk 28]
```diff
@@ -0,0 +1,41 @@
+Generate a compliance report via button.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md:1]:
```suggestion
Generate a compliance report via clicking a button.
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md [hunk 29]
```diff
@@ -0,0 +1,41 @@
...
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.8.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| table | The table of data. |
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md:18]: see my comment in the yml.

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md [hunk 30]
```diff
@@ -0,0 +1,41 @@
...
+| title | The title of the report. |
+| to | The to email |
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md:20]:
```suggestion
| to | The to email address. |
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeLink/CreatePrismaCloudComputeLink.yml [hunk 31]
```diff
@@ -0,0 +1,26 @@
+args:
+- name: type
+  required: true
+  description: The type of WaaS alert.
+- name: imageName
+  required: true
+  description: The image name to investigate alerts for.
+- description: The base url of the Prisma Cloud Compute Instance (https://<prisma-ip>:<prisma-port>)
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeLink/CreatePrismaCloudComputeLink.yml:8]:
```suggestion
- description: The base URL of the Prisma Cloud Compute Instance (https://<prisma-ip>:<prisma-port>).
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeLink/README.md [hunk 32]
```diff
@@ -0,0 +1,47 @@
...
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.5.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| type | The type of WaaS alert. |
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeLink/README.md:18]:
```suggestion
| type | The type of WAAS alert. |
```

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeLink/README.md [hunk 33]
```diff
@@ -0,0 +1,47 @@
...
+| imageName | The image name to investigate alerts for. |
+| baseUrl | The base url of the Prisma Cloud Compute Instance \(https://&amp;lt;prisma-ip&amp;gt;:&amp;lt;prisma-port&amp;gt;\) |
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeLink/README.md:20]:
```suggestion
| baseUrl | The base URL of the Prisma Cloud Compute Instance \(https://&amp;lt;prisma-ip&amp;gt;:&amp;lt;prisma-port&amp;gt;\). |
```

File: Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/README.md [hunk 34]
```diff
@@ -0,0 +1,40 @@
+This is a wrapper around the setIndicators script
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/README.md:1]:
```suggestion
This is a wrapper around the setIndicators script.
```

File: Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/README.md [hunk 35]
```diff
@@ -0,0 +1,40 @@
...
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.8.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| Indicators | The csv list of indicators to tag. |
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/README.md:18]:
```suggestion
| Indicators | The CSV list of indicators to tag. |
```

File: Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.yml [hunk 36]
```diff
@@ -0,0 +1,24 @@
+args:
+- name: Indicators
+  required: true
+  description: The csv list of indicators to tag.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.yml:4]:
```suggestion
  description: The CSV list of indicators to tag.
```

File: Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.yml [hunk 37]
```diff
@@ -0,0 +1,24 @@
...
+- name: Tags
+  required: true
+  description: The tags to add to the indicators.
+commonfields:
+  id: TagIndicatorButton
+  version: -1
+dockerimage: demisto/python3:3.10.10.48392
+enabled: true
+comment: This is a wrapper around the setIndicators script
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.yml:13]:
```suggestion
comment: This is a wrapper around the setIndicators script.
```

**Comment by ShirleyDenkberg** [comment]
@GuyAfik @adi88d Doc review completed.

**Comment by adi88d** [comment]
@nericksen please update the suggestions for the documentation review

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml [hunk 38]
```diff
@@ -0,0 +1,1094 @@
...
+contentitemexportablefields:
+  contentitemfields:
+    packID: ""
+    itemVersion: 1.0.9
+    fromServerVersion: 5.0.0
+    toServerVersion: ""
+    definitionid: ""
+vcShouldKeepItemLegacyProdMachine: false
+name: Prisma Cloud Compute - Audit Alert v2
+description: Default playbook for parsing Prisma Cloud Compute audit alerts
```
[the remark this answers is not in the document]
Comment (nericksen) [reply Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:12]: I suspect this should be a comma :) will update accordingly.

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml (hunk 12 above)
Comment (nericksen) [reply Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml:300]: agreed!

File: Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md (hunk 22 above)
Comment (nericksen) [reply Packs/PrismaCloudCompute/ReleaseNotes/1_3_2.md:27]: added in error.  removed now.

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml (hunk 26 above)
Comment (nericksen) [reply Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml:5]: no this needs to be a `list` of objects. the automation loops through it to create a table.

File: Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md (hunk 29 above)
Comment (nericksen) [reply Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md:18]: I think generally a `table` in xsoar refers to a list of objects.  But maybe I can add a note to that effect to make it more clear?

Something like `The table of data (list of objects).` ??

**Comment by nericksen** [comment]
Documentation feedback implemented.

**Review by adi88d — APPROVED** [review]

**Pull request merged**